forage-android/src/main/java/com/joinforage/forage/android/ecom/services/vault/forage/RosettaPinSubmitter.kt

```diff
@@ -26,10 +26,11 @@ import okhttp3.RequestBody
 import okhttp3.RequestBody.Companion.toRequestBody
 import org.json.JSONObject
 
-internal class ForagePinSubmitter(
+internal class RosettaPinSubmitter(
```

Comment:
Renamed for consistency with POS code and for less ambiguity while we have dual redundancy